### DISCUSSION TITLE
launch server report an error

### DISCUSSION THREAD

**Discussion by zrwsmd** [eclipse-langium/langium#1249] [Q&A]
when i launch langium server ,an error report at 29 line,how  to solve it ,thanks a ahead
![image](https://github.com/eclipse-glsp/glsp/assets/26995040/3daa6294-3b39-45fb-905b-f89235afc210)
![image](https://github.com/eclipse-glsp/glsp/assets/26995040/d17af487-7817-41a8-90be-af202c0e7ceb)

**Comment by msujew** [comment]
I see that the discussion is closed. Have you resolved your issue already? Please post an answer for people that find this discussion and look for a solution for their problem.
